site/ja/blog/new-in-devtools-106/index.md

```diff
@@ -15,39 +15,39 @@ tags:
 draft: true
```

Comment:
```suggestion
```
Remove draft

Reply:
@jecfish Oh, thank you for the suggestion!